bundles/org.eclipse.emfcloud.modelserver.client/.checkstyle

```diff
@@ -1,10 +1,12 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
 <fileset-config file-format-version="1.2.0" simple-config="true" sync-formatter="false">
-  <local-check-config name="org.eclipse.emfcloud.modelserver.checkstyle" location="/org.eclipse.emfcloud.modelserver.codestyle/checkstyle/modelserver-checkstyle.xml" type="project" description="">
-    <additional-data name="protect-config-file" value="true"/>
+  <local-check-config name="EMFcloud Checkstyle" location="https://raw.githubusercontent.com/eclipse-emfcloud/emfcloud/tortmayr/issues/19/codestyle/org.eclipse.emfcloud.checkstyle/src/main/resources/emfcloud-checkstyle.xml" type="remote" description="The central checkstyle configuration for emfcloud">
+    <additional-data name="cache-props-file-location" value="EMFcloud Checkstyle_1618908540765_cache.properties"/>
+    <additional-data name="cache-file" value="true"/>
+    <additional-data name="cache-file-location" value="EMFcloud Checkstyle_1618908540765_cache.xml"/>
   </local-check-config>
-  <fileset name="all" enabled="true" check-config-name="org.eclipse.emfcloud.modelserver.checkstyle" local="true">
+  <fileset name="all" enabled="true" check-config-name="EMFcloud Checkstyle" local="true">
```

Comment:
Should be `EMF.cloud Checkstyle` if possible, just to use consistent naming. If the`.` does not work, this is fine as well.